src/sass/app.scss

```diff
@@ -219,7 +219,8 @@ input[type='submit']::-moz-focus-inner {
 }
 
 button:disabled {
-    background-color: color.scale($arc_rose, $lightness: 15%);
+    background-color: color.scale($arc_rose, $lightness: 45%);
+    cursor: not-allowed;
```

Comment:
makes the disabled buttons properly ghosted, and adds a mousover :no_entry_sign: